src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -311,6 +311,18 @@ var setupSearchButtons = () => {
  * - DOCUMENTATION_OPTIONS.theme_switcher_url
  */
 
+/**
+ * path component of URL
+ */
+var getCurrentUrlPath = () => {
+  if (DOCUMENTATION_OPTIONS.BUILDER == "dirhtml") {
+    return DOCUMENTATION_OPTIONS.pagename == "index"
+      ? `/`
```

Comment:
btw I'm unsure if this would suffice (avoids duplicate `//` in URLs):

```suggestion
      ? ""
```